src/classes/com/sun/tdk/jcov/Merger.java

```diff
@@ -1011,12 +1013,14 @@ private int handleEnv_(EnvHandler opts) throws EnvHandlingException {
         }
 
         template = opts.getValue(DSC_TEMPLATE);
-        Utils.checkFileCanBeNull(template, "template filename", Utils.CheckOptions.FILE_EXISTS, Utils.CheckOptions.FILE_ISFILE, Utils.CheckOptions.FILE_CANREAD);
+        Utils.checkFileCanBeNull(template, "template filename",
+                Utils.CheckOptions.FILE_EXISTS, Utils.CheckOptions.FILE_ISFILE, Utils.CheckOptions.FILE_CANREAD);
 
         addMissing = template == null;
 
         output = opts.getValue(DSC_OUTPUT);
-        Utils.checkFileNotNull(output, "output filename", Utils.CheckOptions.FILE_NOTISDIR, Utils.CheckOptions.FILE_CANWRITE, Utils.CheckOptions.FILE_PARENTEXISTS);
+        Utils.checkFileNotNull(output, "output filename",
```

Comment:
It appears that the set of permissions is the same but in a different order.
It would help if the changset did not contain any formatting changes

Reply:
OK, I just truncated code lines according to the java code style and changed the order of checking to get more natural list of reasons for failure. Do you need to restore the "old" fashion?

Reply:
It's not that important. I am just saying that the purely formatting changes make it harder to make sure that the fix is correct.